kontrakter/siktelseTiltale/1.0/eksempelfiler/siktelseTiltale-eksempel-1.json

```diff
@@ -0,0 +1,154 @@
+{
+  "forsendelse": {
+    "meldingsId": "F16E9255-FDA4-4024-AA23-8A2C62C14497",
+    "sendtTid": "2024-02-28T17:47:36+02:00",
+    "avsender": {
+      "organisasjon": {
+        "organisasjonsnummer": "983998178",
+        "navn": "Sør-Øst politidistrikt"
+      },
+      "person": {
+        "etternavn": "Politiets Datatjeneste"
+      }
+    },
+    "mottakerOrganisasjon": {
+      "organisasjonsnummer": "926723480",
+      "navn": "Agder tingrett"
+    }
+  },
+  "hovedStraffesaksnummer": "15439560",
+  "begjaeringAnmodning": {
+    "kravId": "305B944C-7D1D-491E-AFD3-473F245DABBC"
+  },
+  "siktelseTiltale": {
+    "siktelsesId": "16D161D4-33F4-4651-B1AD-A80917BCB647",
+    "siktelsesType": "SIKTELSE",
+    "poster": [
+      {
+        "nummer": "I",
+        "prinsipalLovbud": {
+          "lovbudKombinertId": "DB1A21B4-5D0B-4B95-9EA4-94306A75DF9B",
+          "lovbud": [
+            {
+              "lovbudId": "{A1D0F6A9-D4C1-4F2D-A69E-039ACB090304}",
+              "lovbudStreng": "Straffeloven § 321",
+              "gjengivelse": "for å ha tatt en gjenstand som tilhører en annen, med forsett om å skaffe seg eller andre en uberettiget vinning ved å selge, forbruke eller på annen måte tilegne seg den."
+            }
+          ]
+        },
+        "straffbareForhold": [
+          {
+            "straffesaksnummer": "15439561",
+            "prinsipalGrunnlag": {
+              "grunnlagTekst": "Mandag 1. mai 2023 kl. 00.00 i Gartneriveien 2 i Kragerø ..."
+            },
+            "basissaker": [
+              {
+                "basissakId": "NzAzODU5MTE0",
+                "personForetak": {
+                  "personRef": "700896090"
+                }
+              }
+            ],
+            "inkluderteForhold": []
+          },
+          {
+            "straffesaksnummer": "15439560",
+            "prinsipalGrunnlag": {
+              "grunnlagTekst": "Lørdag 1. juli 2023 kl. 00.00 i Weiss gate 5D i Kongsberg ..."
+            },
+            "basissaker": [
+              {
+                "basissakId": "NzAzODU5MTEz",
+                "personForetak": {
+                  "personRef": "700896090"
+                }
+              }
+            ],
+            "inkluderteForhold": []
+          }
+        ]
+      }
+    ],
+    "skrevetAv": {
+      "fornavn": "NinaTerese",
+      "etternavn": "Eriksen"
+    }
+  },
+  "personer": [
+    {
+      "internId": "700896090",
+      "navn": {
+        "fornavn": "Vokal",
+        "etternavn": "FABRIKKPIPE"
+      },
+      "identitetsnummer": {
+        "foedselsnummer": "22918197870"
+      },
+      "tilleggsId": [],
+      "kjoenn": "KVINNE",
+      "foedselsdato": "1981-11-22",
+      "statsborgerskap": [
+        {
+          "kode": "NOR",
+          "navn": "Norge"
+        }
+      ],
+      "personAdresse": {
+        "adresse": {
```

Comment:
Fra Calle:
* Adresse. 
	* Postnummer: kun tall - alltid 4 tall? Bør være påkrevd?
	* Hvorfor addreslinjer påkrevd når postnummer og poststed ikke satt
		* Gir det noen gang mening med kun adresselinjer??
	* Kan vi skille på norske og utenladske adresser. OneOff?

Reply:
Laget en norsk og en utlandsk adresse med one of.